docs/conf.py

```diff
@@ -69,23 +69,36 @@
 
 # -- Sphinx Gallery ----------------------------------------------------------
 
+from sunpy_sphinx_theme import PNG_ICON  # noqa
 from sphinx_gallery.sorting import ExplicitOrder  # noqa
 
 sphinx_gallery_conf = {
-    "ignore_pattern": ".*helpers.py",
-    "examples_dirs": "../examples",
+    "abort_on_example_error": False,
+    "backreferences_dir": os.path.join("generated", "modules"),
+    "default_thumb_file": PNG_ICON,
+    "examples_dirs": os.path.join("..", "examples"),
+    "filename_pattern": '^((?!skip_).)*$',
     "gallery_dirs": os.path.join("generated", "gallery"),
-    "subsection_order": ExplicitOrder(["../examples/using_pfsspy",
-                                       "../examples/finding_data",
-                                       "../examples/utils",
-                                       "../examples/pfsspy_info",
-                                       "../examples/testing"]),
-    "reference_url": {"sphinx_gallery": None}
+    "ignore_pattern": "helpers.py",
+    "matplotlib_animations": True,
+    "only_warn_on_example_error": True,
+    "plot_gallery": True,
+    "remove_config_comments": True,
+    "subsection_order": ExplicitOrder([
+        "../examples/using_pfsspy",
+        "../examples/finding_data",
+        "../examples/utils",
+        "../examples/internals",
+        "../examples/testing"
+    ]),
 }
 
 # -- Other options ----------------------------------------------------------
-default_role = 'py:obj'
 
+# JSOC email OS ENV
+# See https://github.com/sunpy/sunpy/wiki/Home:-JSOC
 os.environ["JSOC_EMAIL"] = 'jsoc@sunpy.org'
 
+default_role = 'py:obj'
```

Comment:
Wait what. I added this to the template.

Reply:
Bad merge?